YMTreeMap.swift

```diff
@@ -305,13 +306,13 @@ public extension YMTreeMap {
     }
 }
 
-internal extension YMTreeMap.Rect {
+extension YMTreeMap.Rect {
```

Comment:
Ah, what’s the right way to do this now? Ideally public APIs are minimized because they’re contracts. But this is very minor.

Reply:
I merely removed the `internal` modifier on the extension because it was redundant. The `toSystemRect` property already specifies the access level. This just fixes a compiler warning, there is no actual change to the access level.